persistence/src/commonMain/kotlin/com/wire/kalium/persistence/backup/DatabaseImporter.kt

```diff
@@ -11,7 +11,7 @@ class DatabaseImporterImpl(private val sqlDriver: SqlDriver) : DatabaseImporter
 
     override suspend fun importFromFile(filePath: String, fromOtherClient: Boolean, userDBSecret: UserDBSecret?) {
         val isDBSQLCiphered = userDBSecret != null && userDBSecret.value.isNotEmpty()
-        sqlDriver.execute("""BEGIN""")
+        sqlDriver.newTransaction()
```

Comment:
Question here, just having a look at the API, and this returns a `Transaction`, should we use that returned transaction to execute the queries ? 🤔